homeworks/03-data-structure/src/SimpleBinarySearchTree.test.ts

```diff
@@ -0,0 +1,50 @@
+import BinaryTree, { TreeNode, TraverseType, BinarySearchTree } from './BinaryTree'
+import SimpleBinarySearchTree from './SimpleBinarySearchTree'
+import { N, sort } from './TestHelper'
+
+describe("Simple binary search tree", () => {
```

Comment:
I would expect tests of inherited methods as well.

Reply:
Added. It resulted in copy-pasting tests with small changes, not sure if it's good or not.